src/test/java/edu/hm/hafner/analysis/parser/GhsMultiParserTest.java

```diff
@@ -108,7 +108,18 @@ void issue59118() {
     @Test
     void issue66130() {
         Report warnings = parse("issue66130.log");
-        assertThat(warnings).hasSize(2);
+        assertThat(warnings).hasSize(1);
```

Comment:
Hey @uhafner 

The current regex matches to the second line (with col). This is the error:

```
java.lang.AssertionError: 
Expecting message of:
  <D:/jenkins/src/MyFile.cpp(42,0): -: #381-D: extra ";" ignored

"D:\jenkins\src\MyColTest.cpp", line 42 *(col. 58)*: warning #382-D: extra ";" ignored>
to be:
  <extra ";" ignored
"D:\jenkins\src\MyColTest.cpp", line 42 *(col. 58)*: warning #382-D: extra ";" ignored>
but was:
  <extra ";" ignored

"D:\jenkins\src\MyColTest.cpp", line 42 *(col. 58)*: warning #382-D: extra ";" ignored>
```

Here do you see it wants to assert half part of the first line? `extra ";" ignored` with space of a line is repeated?

Is this is what expected?

It is pretty confusing to me. Unlike the last issue where each line was being asserted.